jujugui/static/gui/src/app/components/env-size-display/env-size-display.js

```diff
@@ -16,8 +16,9 @@ class EnvSizeDisplay extends React.Component {
     const view = e.currentTarget.dataset.view;
     this.props.appState.changeState({
       gui: {
-        machines: view === 'machine' ? '' : null,
-        status: view === 'status' ? '' : null
+        application: view === 'application' ? true : null,
```

Comment:
There is no 'application' state in the gui. The application state is simply the absence of status and machine.